src/formatDate.js

```diff
@@ -51,6 +51,55 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSepar = fromFormat[3];
+  const newSepar = toFormat[3];
+  const currDate = date.split(oldSepar);
+  let day;
+  let month;
+  let year;
+  const newDate = [];
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'DD':
+        day = currDate[i];
+        break;
+      case 'MM':
+        month = currDate[i];
+        break;
+      case 'YYYY':
+        year = currDate[i];
+        break;
+      case 'YY':
+        if (currDate[i] < 30) {
+          year = '20' + currDate[i];
+        } else {
+          year = '19' + currDate[i];
+        }
+        break;
+      default:
+        throw new Error('error');
```

Comment:
error message should be meaningful
```suggestion
        throw new Error('passed format is incorrect');
```